src/autopas/utils/ArrayUtils.h

```diff
@@ -146,4 +147,79 @@ std::enable_if_t<autopas::utils::ArrayUtils::is_container<Container>::value, std
 
   return os;
 }
+
+/**
+ * Given a collection of vectors, redistributes the elements of the vectors so they all have the same (or +1) size.
```

Comment:
Add comment/note briefly describing the distribution process and make it clear that overall ordering is not preserved. E.g. Elements are taken from the ends of too-long vectors and added to the ends of too-short vectors. The overall ordering of elements is not preserved.